Dockerfile

```diff
@@ -7,17 +7,15 @@ RUN lein uberjar
 
 # -----------------------------------------------------------------------------
 
-from openjdk:11
+FROM adoptopenjdk/openjdk11:alpine-jre
 
-RUN groupadd -r commentator && useradd -r -s /bin/false -g commentator commentator
+RUN addgroup -S commentator && \
+    adduser -s /bin/false -G commentator -S commentator && \
+    apk add --update-cache git && \ 
```

Comment:
As you said on twitter, git can be removed :D

Reply:
Done :)